sdk/identity/azure-identity/test/live/client_secret_credential.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/identity/client_secret_credential.hpp>
+
+#include <gtest/gtest.h>
+
+#include <chrono>
+#include <cstdlib>
+
+inline std::string GetEnv(std::string const& name)
```

Comment:
@ahsonkhan This looks like a more complete way to get the values from the env. I think we can re-use this or duplicate it for keyvault tests.